styles/brand-tokens.css

```
@@ -46,111 +52,180 @@
+  --mg-100: oklch(0.925 0.058 320);
+  --mg-200: oklch(0.86 0.105 320);
+  --mg-300: oklch(0.775 0.16 320);
+  --mg-400: oklch(0.68 0.215 320);
+  --mg-500: oklch(0.585 0.255 320);
+  --mg-600: oklch(0.5 0.24 320);
+  --mg-700: oklch(0.415 0.199 320);
+  --mg-800: oklch(0.32 0.155 320);
+  --mg-900: oklch(0.235 0.11 320);
+
+  /* Type, space, radius, motion. */
+  --font-sans: Inter, system-ui, -apple-system, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Inter" to be "inter" (value-keyword-case)

(value-keyword-case)